getting-started.md

```diff
@@ -0,0 +1,64 @@
+# Getting started
+
+The TiKV client is a Rust library (crate). To use this crate in your project, add the following dependencies to your `Cargo.toml`:
+
+```toml
+[dependencies]
+tikv-client = "0.1"
```

Comment:
Shall we keep it 0.0.99 and change it when we publish 0.1?

Reply:
I hope it won't take long, and we use 0.0.99 in the README